js/components/user/profile.vue

```diff
@@ -20,13 +20,13 @@
 
 <template>
 <div>
-<box :title="title" icon="user" boxclass="user-profile-widget">
-    <h3>{{user.fullname}}</h3>
+<box :title="user.fullname" icon="user" boxclass="user-profile-widget">
         <div class="profile-body">
             <image-button :src="user | avatar_url 100" :size="100" class="avatar-button"
                 :endpoint="endpoint" :editable="can_edit">
             </image-button>
         <div v-markdown="user.about"></div>
+        <h4>{{user.email}}</h4>
```

Comment:
`v-if`?

Reply:
Well as the user seeing this view is either an admin or himself, the api will always return the user's email.

Reply:
Nope :-) you can access any user profile page in the admin as long as you're logged in (no need to be an admin or self).